components/admin_console/admin_definition.jsx

```diff
@@ -5769,6 +5769,43 @@ const AdminDefinition = {
             },
         },
     },
+    products: {
+        icon: (
+            <CogOutlineIcon
+                size={16}
+                className={'category-icon fa'}
+                color={'currentColor'}
+            />
+        ),
+        sectionTitle: t('admin.sidebar.products'),
+        sectionTitleDefault: 'Products',
+        isHidden: it.any(
+            it.configIsFalse('FeatureFlags', 'BoardsProduct'),
```

Comment:
@hmhealey - It seems like our test case suite somehow doesn't like `it.configIsFalse` under the main products section. But the code works, and the section indeed gets hidden. If I move the function under `settings [`, the tests pass.

I can't figure out what needs to change in the tests for this to pass. Any pointers would be very appreciated.